snippets/csharp/VS_Snippets_CLR/FInfo Class/CS/finfo class.cs

```diff
@@ -31,23 +31,23 @@ public static void Main()
         try 
         {
             string path2 = Path.GetTempFileName();
-            FileInfo fi2 = new FileInfo(path2);
+            var fi2 = new FileInfo(path2);
 
-            //Ensure that the target does not exist.
+            // Ensure that the target does not exist.
             fi2.Delete();
 
-            //Copy the file.
+            // Copy the file.
             fi1.CopyTo(path2);
-            Console.WriteLine("{0} was copied to {1}.", path, path2);
+            Console.WriteLine($"{path1} was copied to {path2}.");
```

Comment:
There;s a typo in your change:

```suggestion
            Console.WriteLine($"{path} was copied to {path2}.");
```